### PR TITLE
[REACTOS] Fix GetTokenInformation() usage

### PR DESCRIPTION
**Pull request by SergeGautherie**
…return value checks

### CODE REVIEW

**Comment by Extravert-ir** [comment]
You're inverting the logic here. Can you elaborate why previous code is wrong and we should use your version.

And please, do similar changes in one pull request!

**Comment by SergeGautherie** [comment]
> why previous code is wrong

Obviously, current code errors out as soon as `GetTokenInformation()` fails, so `GetLastError()` is not checked. In other words, the rest of the function is dead code.

> And please, do similar changes in one pull request!

Merged 3 previous PRs.

File: dll/win32/netid/netid.c [hunk 1]
```diff
@@ -180,13 +180,15 @@ IsUserAdmin(VOID)
         goto done;
 
     dwSize = 0;
-    GetTokenInformation(hToken,
-                        TokenGroups,
-                        NULL,
-                        0,
-                        &dwSize);
-    if (dwSize == 0)
+    if (GetTokenInformation(hToken,
```
Comment (Fraizeraust) [inline dll/win32/netid/netid.c:183]: `GetTokenInformation` returns a BOOL value. With your change the code literally hits the check when this function successfully retrieves details about the token....

Comment (Extravert-ir) [reply dll/win32/netid/netid.c:183]: This is how it supposed to be used: first you call `GetTokenInformation` with NULL buffer and expect it to return `ERROR_INSUFFICIENT_BUFFER`. Then you make a proper call

Comment (Extravert-ir) [inline dll/win32/netid/netid.c:183]: While you're here, please format this condition as others.

**Pull request merged**